src/components/Modals/ConfirmationModal/index.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import {
+  Dialog,
+  DialogContent,
+  DialogActions,
+  Typography,
+  DialogTitle,
+} from "@mui/material";
+import "./ConfirmationModal.css";
+import ModalActionButton, { Variant } from "./ModalActionButton";
+import closeIcon from "../../../img/icons/close-icon.svg";
+
+type Props = {
+  showed: boolean;
+  title: string;
+  description: string;
+  hideModal?: () => void;
```

Comment:
Is it possible to leave "showed" as an internal detail of the ConfirmationModal and expose only method for opening and closing the modal?

On top of that I would bound closeModal as default action for cancel button (we can add handleCancelation later there is a case that require customisation of the cancel button)